clusters/app.ci/supplemental-ci-images/envoyproxy/envoyproxy.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: envoyproxy
+  namespace: ci
+spec:
+  tags:
+  - name: "1.25.3"
+---
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  namespace: ci
+  name: envoyproxy
+spec:
+  successfulBuildsHistoryLimit: 1
+  failedBuildsHistoryLimit: 2
+  output:
+    to:
+      kind: ImageStreamTag
+      name: envoyproxy:tp-1.25.3
+  source:
+    dockerfile: |
+      FROM quay.io/centos/centos:stream9
+      LABEL maintainer="jguzik@redhat.com"
+      RUN curl -L -o /bin/envoy https://github.com/envoyproxy/envoy/releases/download/v1.25.3/envoy-1.25.3-linux-x86_64 && chmod +x /bin/envoy
```

Comment:
How can we maintain this for different architectures?

Reply:
why we should? This is only deployment for app.ci